azure/terraform/README.md

```diff
@@ -31,10 +31,22 @@ This is used to help generate unique resource names for:
 * Azure storage account name
 Note: these resources must be globally unique across all of Azure.
 
+The template also provides the `deploy_airflow` and `deploy_argo` flags. These are booleans that specify if airflow or argo will be deployed in the kubernetes cluster along with metaflow related services. 
+
 Next, apply the `infra` module (creates Azure cloud resources only).
 
     terraform apply -target="module.infra"
 
+### Airflow Deployment with Azure
+If `deploy_airflow` is set to true, then the `infra` module will create one more storage blob-container named `airflow-logs` and will also provide blob-container R/W permissions to the service principle. We create this extra blob-container because Airflow expects the default blob-container name to be `airflow-logs` and provides [no way to configure that from top level](https://github.com/apache/airflow/blob/b19ccf8ead027d9eaf53b33305be5873f2711699/airflow/config_templates/airflow_local_settings.py#L241) (At version 2.3.3, can change in the future). 
```

Comment:
May be just link to the Airflow code reference directly?  I guess no need to be so explicit about "no way to configure, and at version 2.3.3, can change, etc".